frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -242,6 +255,24 @@ const BaseGroupExpandedComponent: React.FunctionComponent<BaseGroupExpandedProps
     >
       <Layer id={GROUPS_LAYER}>
         <g ref={refs} onContextMenu={onContextMenu} onClick={onSelect} className={innerGroupClassName}>
+          {!!isFocused && (
+            <path
+              ref={outlineRef as any}
+              className={styles.topologyGroupBackground}
+              d={pathRef.current}
+              style={
+                !!isFocused
+                  ? {
+                      stroke: ColorFocus,
+                      strokeOpacity: OverlayOpacity,
+                      animationDuration: '1s',
+                      animationName: flashAnimation,
+                      animationIterationCount: 3
+                    }
+                  : {}
+              }
```

Comment:
```suggestion
              style={{
                stroke: ColorFocus,
                strokeOpacity: OverlayOpacity,
                animationDuration: '1s',
                animationName: flashAnimation,
                animationIterationCount: 3
              }}
```

The `isFocused` is already guaranteed, above.